src/generic_ui/scripts/ui.spec.ts

```diff
@@ -76,7 +76,6 @@ describe('UI.UserInterface', () => {
         userId: userId,
         name: userName,
         imageData: 'testImageData',
```

Comment:
So, personally, I actually like the trailing commas (when you add a new field later, you don't have to touch the last line, which makes sense to me).

Is there a reason not to use them?